Dockerfile

```diff
@@ -20,9 +20,20 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# FROM gcr.io/distroless/static:nonroot
+FROM alpine:3.7
```

Comment:
Could try `FROM openshift/origin-release:golang-1.15 as builder` it may have git there. 

https://github.com/openshift/release/blob/65f3976811b3a4fe333339b87a5c8b766c60b775/projects/origin-release/golang-1.15/Dockerfile